### DISCUSSION TITLE
Tasmota v12.4.0: Shows Temperature on ShellyPlug S but not on NousA1T

### DISCUSSION THREAD

**Discussion by pdfkungfoo** [arendst/Tasmota#18099] [Q&A] [answered]
I'm new to *Tasmota* and new to smart plugs. After acquiring a *Nous A1T* (coming pre-flashed with Tasmota 12.4.0) and a *Shelly Plug S* (which I re-flashed with the same Tasmota version), I observe a small difference:
- for *Shelly Plug S* the WebUI displays a value for the device temperature
- for the *Nous A1T* it doesn't

<img width="216" alt="image" src="https://user-images.githubusercontent.com/4747960/222708671-f6407376-89b1-4a13-9b9a-c746784df311.png"> <img width="188" alt="image" src="https://user-images.githubusercontent.com/4747960/222708512-7f4f9178-3e37-4739-9aff-0e909630c004.png">

What is the reason for this?
- The Nous does not have a temperature sensor in hardware?
- The template used for the Nous is incomplete? Tasmota software is not (yet) feature-complete to get everything out of the Nous?

Is there anything I can do to get the temperature metering from the Nous A1T?

**Comment by barbudor** [comment, marked as answer]
There is no temperature sensor in the Nous A1T
Beside as for every device which as power measurent, the ESP is not isolated from the mains so we totally discourage to tinker with such products in order to add sensors

In the Shelly, the temperature sensor is a security which would switch power of in case of overheat
So don't expect it to measure the temperature of the room

So no temperature measures on the Nous A1T

**Reply by pdfkungfoo** [reply]
Thank you, @barbudor!
